src/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ruby:2.5.1
+
+RUN mkdir -p /usr/src/ruby_first
+WORKDIR /usr/src/ruby_first
+
+ENV HOME=/usr/src/ruby_first PATH=/usr/src/ruby_first/bin:$PATH
+
+RUN set -ex \
+  && for key in \
+    9554F04D7259F04124DE6B476D5A82AC7E37093B \
+    94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
+    0034A06D9D9B0064CE8ADF6BF1747F4AD2306D93 \
+    FD3A5288F042B6850C66B31F09FE44734EB7990E \
+    71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
+    DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+    C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+  ; do \
+    gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
+  done \
+  && export NPM_CONFIG_LOGLEVEL=info \
+  && export NODE_VERSION=6.3.1 \
+  && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" \
+  && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
+  && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
+  && grep " node-v$NODE_VERSION-linux-x64.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
+  && tar -xJf "node-v$NODE_VERSION-linux-x64.tar.xz" -C /usr/local --strip-components=1 \
+  && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
+
+COPY /ruby_first/Gemfile* /usr/src/ruby_first/
+RUN bundle install
+
+COPY /ruby_first /usr/src/ruby_first/
```

Comment:
You need to use `ADD` command instead of `COPY`

Reply:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Reply:
Looks like `copy` is better.